test/tools/ossfuzz/abiV2FuzzerCommon.h

```diff
@@ -0,0 +1,62 @@
+#pragma once
+
+#include <libsolidity/interface/CompilerStack.h>
+#include <libyul/AssemblyStack.h>
+#include <liblangutil/Exceptions.h>
+#include <liblangutil/SourceReferenceFormatter.h>
+#include <libdevcore/Keccak256.h>
+
+namespace dev
+{
+namespace test
+{
+namespace abiv2fuzzer
+{
+class SolidityExecutionFramework
+{
+public:
+	SolidityExecutionFramework();
+	SolidityExecutionFramework(langutil::EVMVersion _evmVersion);
+
+	Json::Value getMethodIdentifiers()
+	{
+		return m_compiler.methodIdentifiers(m_compiler.lastContractName());
+	}
+
+	dev::bytes compileContract(
```

Comment:
I think this could easily be a function in an anonymous namespace inside abiV2Fuzzer.cpp

Reply:
I put it in `abiV2FuzzerCommon.h` so that something like `abiV2ProtoFuzzer.cpp` could also include it.